src/SortableContainer/index.js

```diff
@@ -513,15 +513,15 @@ export default function sortableContainer(WrappedComponent, config = {withRef: f
 
     updatePosition(e) {
       const {lockAxis, lockToContainerEdges} = this.props;
-      
+
       const offset = this.getOffset(e);
       const translate = {
         x: offset.x - this.initialOffset.x,
         y: offset.y - this.initialOffset.y,
       };
       // Adjust for window scroll
-      translate.y -= (window.scrollY - this.initialWindowScroll.top);
-      translate.x -= (window.scrollX - this.initialWindowScroll.left);
+      translate.y -= (window.scrollY - this.initialWindowScroll.top) || window.pageYOffset;
+      translate.x -= (window.scrollX - this.initialWindowScroll.left) || window.pageXOffset;
```

Comment:
I've updated this pr with the correct assignment of our initial scroll values so our diff is accurate when using IE 11.